apps/src/lib/levelbuilder/lesson-editor/ActivitySectionCardButtons.jsx

```diff
@@ -164,66 +203,39 @@ class ActivitySectionCardButtons extends Component {
       <div>
         <div style={styles.bottomControls}>
           <span>
-            <button
-              onClick={this.handleOpenAddLevel}
-              className="btn uitest-open-add-level-button"
-              style={styles.addButton}
-              type="button"
-            >
-              <i style={{marginRight: 7}} className="fa fa-plus-circle" />
-              Level
-            </button>
+            <AddButton
+              className="uitest-open-add-level-button"
+              displayText="Level"
+              handler={this.handleOpenAddLevel}
+            />
             {this.props.hasLessonPlan && (
               <span>
-                <button
-                  onMouseDown={this.handleOpenAddTip}
-                  className="btn"
-                  style={styles.addButton}
-                  type="button"
-                >
-                  <i style={{marginRight: 7}} className="fa fa-plus-circle" />
-                  Callout
-                </button>
-                <button
-                  onMouseDown={this.handleOpenAddResource}
-                  className="btn"
-                  style={styles.addButton}
-                  type="button"
-                >
-                  <i style={{marginRight: 7}} className="fa fa-plus-circle" />
-                  Resource
-                </button>
+                <AddButton
+                  displayText="Callout"
+                  handler={this.handleOpenAddTip}
+                />
+                <AddButton
+                  displayText="Doc Link"
```

Comment:
Wondering about the name here. Trying to think what will be most clear to a new user. @tess323 do you have thoughts?

Reply:
Agreed - I would go either "Documentation" or "Code Doc".  That matches resources which are also basically a link.